### PR TITLE
Fix sentinel reactive test rule resource name

### LINKED ISSUE #3205: SCA Class of ReactiveSentinelCircuitBreakerIntegrationTest CI/CD Failed
代码提交到github仓库进行自动化检测时偶发失败情况

### 堆栈异常
```
2023-03-14 02:56:23.854  INFO 1952 --- [o-auto-1-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 31 ms
com.alibaba.csp.sentinel.slots.block.degrade.DegradeException
com.alibaba.csp.sentinel.slots.block.degrade.DegradeException
com.alibaba.csp.sentinel.slots.block.degrade.DegradeException
com.alibaba.csp.sentinel.slots.block.degrade.DegradeException
com.alibaba.csp.sentinel.slots.block.degrade.DegradeException
2023-03-14 02:56:32.892  INFO 1952 --- [           main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 15.187 s <<< FAILURE! - in com.alibaba.cloud.circuitbreaker.sentinel.ReactiveSentinelCircuitBreakerIntegrationTest
Error:  test(com.alibaba.cloud.circuitbreaker.sentinel.ReactiveSentinelCircuitBreakerIntegrationTest)  Time elapsed: 8.178 s  <<< FAILURE!
java.lang.AssertionError: expectation "expectNext(flux_fallback)" failed (expected value: flux_fallback; actual value: slowflux)

[INFO] Running com.alibaba.cloud.circuitbreaker.sentinel.ReactiveSentinelCircuitBreakerTest
02:56:34.106 [main] DEBUG reactor.util.Loggers$LoggerFactory - Using Slf4j logging framework
INFO: Sentinel log output type is: file
INFO: Sentinel log charset is: utf-8
INFO: Sentinel log base directory is: /home/runner/logs/csp/
INFO: Sentinel log name use pid is: false
INFO: Sentinel log level is: INFO
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.787 s - in com.alibaba.cloud.circuitbreaker.sentinel.ReactiveSentinelCircuitBreakerTest
[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    ReactiveSentinelCircuitBreakerIntegrationTest.test expectation "expectNext(flux_fallback)" failed (expected value: flux_fallback; actual value: slowflux)
[INFO] 
Error:  Tests run: 12, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
```

### PR DESCRIPTION
**Pull request by ruansheng8**
**Don't merge this PR**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -108,10 +108,10 @@
 
     <modules>
         <module>spring-cloud-alibaba-dependencies</module>
-        <module>spring-cloud-alibaba-examples</module>
-        <module>spring-cloud-alibaba-docs</module>
+<!--        <module>spring-cloud-alibaba-examples</module>-->
```
Comment (steverao) [inline pom.xml:111]: Why comment the relevant line?

File: spring-cloud-alibaba-starters/pom.xml [hunk 2]
```diff
@@ -15,22 +15,22 @@
     <description>Spring Cloud Alibaba Starters</description>
     
     <modules>
-        <module>spring-cloud-starter-alibaba-nacos-config</module>
-        <module>spring-cloud-starter-alibaba-nacos-discovery</module>
-        <module>spring-cloud-starter-alibaba-seata</module>
-        <module>spring-cloud-starter-stream-rocketmq</module>
-        <module>spring-cloud-starter-bus-rocketmq</module>
-        <module>spring-cloud-starter-alibaba-sidecar</module>
+<!--        <module>spring-cloud-starter-alibaba-nacos-config</module>-->
```
Comment (steverao) [inline spring-cloud-alibaba-starters/pom.xml:18]: Same problem.

File: spring-cloud-alibaba-starters/spring-cloud-circuitbreaker-sentinel/src/test/java/com/alibaba/cloud/circuitbreaker/sentinel/ReactiveSentinelCircuitBreakerIntegrationTest.java [hunk 3]
```diff
@@ -62,95 +64,69 @@ public class ReactiveSentinelCircuitBreakerIntegrationTest {
...
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-		StepVerifier.create(service.slow()).expectNext("slow").verifyComplete();
-
-		// Then in the next 5s, the fallback method should be called.
-		for (int i = 0; i < 5; i++) {
-			StepVerifier.create(service.slow()).expectNext("fallback").verifyComplete();
-			Thread.sleep(900);
+		int idx = this.idx.getAndIncrement();
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-circuitbreaker-sentinel/src/test/java/com/alibaba/cloud/circuitbreaker/sentinel/ReactiveSentinelCircuitBreakerIntegrationTest.java:77]: Please add some comment to explain your logic here, it's not easy to understand the logic in short time.

Comment (ruansheng8) [reply spring-cloud-alibaba-starters/spring-cloud-circuitbreaker-sentinel/src/test/java/com/alibaba/cloud/circuitbreaker/sentinel/ReactiveSentinelCircuitBreakerIntegrationTest.java:77]: This PR is just a test

File: pom.xml (hunk 1 above)
Comment (ruansheng8) [reply pom.xml:111]: Remove excess dependencies to test the relevant code, I will restore these dependencies in the final version

**Pull request closed without merging**